src/components/validations/validation-icon.component.js

```diff
@@ -31,23 +40,27 @@ const ValidationIcon = ({
     };
   }
 
+  const validationType = getValidationType({ error, warning, info });
+
+  const validationMessage = error || warning || info;
```

Comment:
Isn't there a chance that this might be a boolean?

Reply:
This could mess up the `aria` props too.

Reply:
I fixed it now it's no longer possible to render the validation icon when validation props are passed as booleans

Reply:
Awesome thanks 👍